milena/Makefile.am

```diff
@@ -40,7 +40,7 @@ tests:
 
 # Regen files recursively.
 include $(top_srcdir)/build-aux/regen-recursive.mk
-REGEN_SUBDIRS += tests
+REGEN_SUBDIRS += doc tests
```

Comment:
Was this intentional? Didn't we disable building the documentation?

Reply:
Yes, but this change is correct: First we disabled by patching the automake files in our clone, then Edwin made the respective permanent fix ee02495a3986a359b4a7e82b94e211b48c5ca196 upstream (by adding a configure switch `ENABLE_DOC` instead). We see the result of the merge here.